src/mutations/member.test.ts

```diff
@@ -205,6 +205,44 @@ describe('Member Mutations', () => {
       expect(newData).toMatchObject(response);
     });
 
+    it(`Successfully edit member's username and ensures trimming id = ${member.id}`, async () => {
+      const response = { ...member, name: newMember.name };
+      // set random data in cache
+      queryClient.setQueryData(memberKeys.current().content, member);
+      const endpoints = [
+        {
+          response,
+          method: HttpMethod.Patch,
+          route,
+        },
+      ];
+      const patchSpy = vi.spyOn(axios, 'patch');
+      const mockedMutation = await mockMutation({
+        mutation,
+        wrapper,
+        endpoints,
+      });
+      const newUsernameWithTrailingSpace = 'newname  ';
```

Comment:
In order to make the response match the input you provide, use this in the reponse definition. (the trimmed version).